application-mocca-calendar-ui/src/main/resources/MoccaCalendar/WebHome.xml

```diff
@@ -50,9 +50,11 @@
 == $services.localization.render('MoccaCalendar.calendar.viewall') ==
 
 #if($hasCreatePage)
-{{html}}&lt;span class="buttonwrapper"&gt;&lt;a class="button" href="$doc.getURL('create','template=MoccaCalendar.MoccaCalendarTemplate')"&gt;$escapetool.xml($services.localization.render('MoccaCalendar.calendar.create'))&lt;/a&gt;&lt;/span&gt;{{/html}}
+{{html}}&lt;span class="buttonwrapper"&gt;&lt;a class="button" href="$doc.getURL('create','template=MoccaCalendar.MoccaCalendarTemplate')"&gt;$escapetool.xml($services.localization.render('MoccaCalendar.calendar.create'))&lt;/a&gt;&lt;/span&gt;{{/html}} ##
+#end
+#if($hasCreatePage)
+{{html}}&lt;span class="buttonwrapper"&gt;&lt;a class="button" href="$doc.getURL('create','template=MoccaCalendar.MoccaCalendarSubscribeTemplate')"&gt;$escapetool.xml($services.localization.render('MoccaCalendar.calendar.subscribe'))&lt;/a&gt;&lt;/span&gt;{{/html}}
```

Comment:
No need to create another if here. You can render the span inside the same html macro as the other one.